cmd/kpod/stats.go

```diff
@@ -46,8 +40,8 @@ var (
 			Usage: "pretty-print container statistics using a Go template",
 		},
 		cli.BoolFlag{
-			Name:  "json",
-			Usage: "output container statistics in json format",
+			Name:  "no-reset",
```

Comment:
Do we no longer support --json?

Reply:
json can be passed as --format like the rest of kpod.

Reply:
Ok